src/api/app/views/webui/feeds/notifications.rss.builder

```diff
@@ -0,0 +1,26 @@
+xml.rss version: '2.0' do
+  xml.channel do
+    xml.title "#{@user.realname} (#{@user.login}) notifications"
+    xml.description "Event notifications from #{@configuration['title']}"
+    xml.link url_for only_path: false, controller: 'main', action: 'index'
+    xml.language "en"
+    xml.pubDate Time.now
+    xml.generator @configuration['title']
+
+    @notifications.each do |notification|
+      xml.item do
+        xml.title notification.event.subject
+        # We render the event_mailer view to 'description'
+        xml.description ApplicationController.renderer.new.render(
```

Comment:
Can you set `@host` and `@configuration` in the controller? Either that or change @host/@configuration to be local variables in the views so you can use the `locals:` option?